woohoo.rb

```diff
@@ -0,0 +1,5 @@
+(1..20).reduce(&:+)
+(1..20).inject(&:+)
+
+'super long linesuper long linesuper long linesuper long linesuper long linesuper long linesuper long linesuper long linesuper long linesuper long linesuper long linesuper long line'
```

Comment:
Literal `'super long linesuper long linesuper long linesuper long linesuper long linesuper long linesuper long linesuper long linesuper long linesuper long linesuper long linesuper long line'` used in void context.<br>Line is too long. [182/100]